nomenclature/definitions/variable/emissions/emissions.yaml

```
@@ -290,6 +290,42 @@ Emissions|CO2|AFOLU:
                 (IPCC category 3)
    unit: Mt CO2/yr
 
+Emmissions|CO2|Electricity:
```

Emmissions|CO2|Electricity:
```suggestion
Emissions|CO2|Electricity:
```